src/test/java/uk/gov/hmcts/darts/cases/service/impl/CloseOldCasesProcessorImplTest.java

```diff
@@ -94,16 +92,17 @@ void closeCases() {
         when(caseService.getCourtCaseById(1)).thenReturn(courtCase);
 
         CaseRetentionEntity caseRetention = createRetentionEntity(courtCase, userAccountEntity);
-        when(retentionApi.createRetention(any(), any(), any(), any(), any())).thenReturn(caseRetention);
+        when(retentionApi.createRetention(any(), any(), any(), any(), any(), any())).thenReturn(caseRetention);
         assertFalse(courtCase.getClosed());
-        assertNull(courtCase.getRetConfUpdatedTs());
+
+        when(retentionApi.updateCourtCaseConfidenceAttributesForRetention(any(), eq(RetentionConfidenceCategoryEnum.AGED_CASE)))
+            .thenReturn(courtCase);
 
         // when
         closeOldCasesProcessor.closeCases(2);
 
         // then
         assertTrue(courtCase.getClosed());
-        assertEquals(CURRENT_DATE_TIME, courtCase.getRetConfUpdatedTs());
```

Comment:
Author note: The class under test no longer has responsibility for setting this attribute, therefore the assertion is removed here. Dedicated unit tests exist in `RetentionServiceImplTest` to verify this attribute is set correctly.